descriptors/ModuleDescriptor-template.json

```diff
@@ -4,7 +4,7 @@
   "provides": [
     {
       "id": "users",
-      "version": "15.5",
+      "version": "16.0",
```

Comment:
Do we know if ui-users is the only module that uses this API's facets?

Reply:
For the centralised modules, I believe so.

I did a search for facets in github and checked all of the code that referenced the term. The only one I found that refers to them users facets is the code that has now been removed from ui-users